akka-stream/src/main/scala/akka/stream/IOResult.scala

```diff
@@ -1,7 +1,7 @@
 /**
  * Copyright (C) 2016 Typesafe Inc. <http://www.typesafe.com>
  */
-package akka.stream.io
+package akka.stream
```

Comment:
I don't mind but would like to understand the reason behind this move - why not in `io`?